### PR TITLE
gha workflows build and push arm64 images

### PR DESCRIPTION
**Pull request by markpash**
Not sure how to test to see if this will work correctly, but hopefully it should.

### CODE REVIEW

**Comment by markpash** [comment]
@aanm please take a look when you have the time, thanks. ❤️

File: .github/workflows/docker-ci.yaml [hunk 1]
```diff
@@ -34,6 +34,9 @@ jobs:
           ref: ${{ steps.tag.outputs.tag }}
           persist-credentials: false
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```
Comment (aanm) [inline .github/workflows/docker-ci.yaml:38]:
```suggestion
        uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3.0.0
```

File: .github/workflows/docker-release.yaml [hunk 2]
```diff
@@ -29,6 +29,9 @@ jobs:
           ref: ${{ steps.tag.outputs.tag }}
           persist-credentials: false
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```
Comment (aanm) [inline .github/workflows/docker-release.yaml:33]:
```suggestion
        uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3.0.0
```

**Review by aanm — CHANGES_REQUESTED** [review]
@markpash thanks, just one small suggestion.

**Review by aanm — APPROVED** [review]

**Comment by markpash** [comment]
@aanm tested it, works fine on my mac now.

**Pull request merged**